lib/takeaway.rb

```diff
@@ -0,0 +1,21 @@
+class Takeaway
+
+  attr_reader :menu
+
+  def initialize
+    @menu = {"starter" => 5, "mains" => 10, "desert" => 3}
+  end
+
+  def show_dishes
+    print "starter : 5\nmains : 10\ndesert : 3\n"
+  end
+
+  def order(*dishes)
+    dishes.pop
+    order_hash = Hash[*dishes]
+    order_hash.each do |k, v|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)